ui/src/utils/array-utils.ts

```diff
@@ -1,5 +1,6 @@
 import { TimeStamp } from 'common/common-model';
 import { expectDefined } from 'utils/type-utils';
+import equal from 'fast-deep-equal';
```

Comment:
Meillä on ollut `object-utils.ts`-tiedostossa funktio nimeltä `objectEquals`, joka on käytännössä tehnyt `JSON.stringify`-vertailun. Sitä ei ole käytetty kauhean monessa paikassa, mutta kuitenkin muutamassa. Jos/kun tämä ulkoinen kirjasto tekee vertailua jotenkin fiksummin ja/tai tehokkaammin, niin voisi harkita `objectEquals`:in heivaamista kokonaan pois koodista ja sen muuttamista siten että se on vain läpikutsu `fast-deep-equal`:in `equal`-funktioon

Reply:
Suosisin mielummin läpikutsua meidän funkkarin kautta. Se sentään mahdollistaa kirjaston helpon vaihtamisen tai jopa toteuttamisen itse jos tuo kirjasto alka tuntumaan niheältä. Joku leftpad-skenaario on aika ärsyttävä jos se depsu näkyy pitkin poikin koodia.